src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -65,7 +65,7 @@
                 {:opacity bg-opacity}
                 {:flex 1 :background-color colors/neutral-100-opa-70})}]]
      ;; sheet
-     [gesture/gesture-detector {:gesture sheet-gesture}
+     [gesture/gesture-detector {:gesture sheet-gesture}  ;; FIXME: This gesture handler is interfering with scrollview on android.
```

Comment:
we use TODO and it should include a github link to that issue

Reply:
It's just for me to fix before moving the PR to ready. I will use TODO for adding future tasks 👍